ext/standard/tests/math/bindec_variation1.phpt

```diff
@@ -77,7 +77,9 @@ foreach($inputs as $input) {
 		var_dump(bindec($input));
 	} catch (TypeError $e) {
 		echo $e->getMessage(), "\n";
-	}
+	} catch (InvalidArgumentException $e) {
+        echo "INVALID\n";
```

Comment:
This is just a nitpic, but I'd prefer if the error message would be echoed (similar to the `TypeError` above)

Reply:
Agree with @kocsismate